wiki/Main_Page/en.md

```diff
@@ -102,18 +102,18 @@ Social interaction is one of main things that fuel osu! players all around the w
 
 [Projects](/wiki/Community/Projects): [Featured Artists](/wiki/Featured_Artists) • [osu! community meetings](/wiki/Community/osu!_community_meetings) • [osu!academy](/wiki/Community/Video_series/osu!academy) • [osu!mapping](/wiki/Community/Video_series/osu!mapping) • [osu!talk](/wiki/Community/Video_series/osu!talk)
 
+Organisations: [osu! UCI](/wiki/Community/Organisations/osu!_UCI)
+
 </div>
 <div class="wiki-main-page-panel">
 
-# People
+# [People](/wiki/People)
 
 osu! wouldn't have been possible without many users helping with development, maintenance, and community management: their efforts, time, and dedication make the game live and prosper.
 
 [The Team](/wiki/People/The_Team): [Developers](/wiki/People/The_Team/Developers) • [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) • [Support Team](/wiki/People/The_Team/Support_Team) • [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) • [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) • [osu! Alumni](/wiki/People/The_Team/osu!_Alumni) • [Project Loved Team](/wiki/People/The_Team/Project_Loved_Team)
 
-Organisations: [osu! UCI](/wiki/Community/Organisations/osu!_UCI)
-
-[Community Contributors](/wiki/People/Community_Contributors) • [Users with unique titles](/wiki/People/Users_with_unique_titles)
+[Community Contributors](/wiki/People/Community_Contributors) • [Performance Points Committee](/wiki/People/Performance_Points_Committee) • [Tournament Committee](/wiki/People/Tournament_Committee) • [Users with unique titles](/wiki/People/Users_with_unique_titles)
```

Comment:
@TicClick opinion?

some translators seem to be sorting index pages as they go recently (i think), though this page obv doesn't have translators with their own sortings. maybe the main page should have fixed ordering regardless of translations (it's easier to navigate sorted lists, but the main page is just a big cloud of links).

either way i'd keep the sorting on the translations equal to the english ver., but do you have any opinions on the ordering on this?

Reply:
for regular index pages, such as one for disambiguation articles, it makes sense to sort the articles lexicographically, as it's more natural and they don't have any other clear ordering.

last time I worked on the main page, I've tried to order pages by their "importance", or "significance", so I'd say make the translations follow the English page

Reply:
alright so which do you think are the important ones here? maybe `Community Contributors` --> `Users with unique titles` --> `Tournament Committee` --> `Performance Points Committee`?

Reply:
anything will do, your suggestion also works (I'd make it roughly the same)